src/parser/expression/constant_value_expression.cpp

```diff
@@ -250,6 +251,59 @@ std::string ConstantValueExpression::ToString() const {
   }
 }
 
+ConstantValueExpression ConstantValueExpression::FromString(const std::string &val, uint64_t type_id) {
```

Comment:
Is this used anywhere? Also the parameter type should be `TypeId`. Force the caller to do a `static_cast` if necessary. Doesn't make sense to allow it to be a `uint64_t` and then cast it on the first line.

Reply:
Resolved in the latest push.